### DISCUSSION TITLE
CentOS Stream

### DISCUSSION THREAD

**Discussion by themew** [magenx/Magento-2-server-installation#25] [Q&A] [answered]
We converted our magenx installation from CentOS 8 to CentOS 8 stream with no issues.  Nothing broken.
Do you recommend keeping the install on CentOS 8 stream or waiting until you've created a Debian/Ubuntu script?

**Comment by magenx** [comment, marked as answer]
right now rewriting into some universal setup for `Ubuntu 20.04`; `Debian 10,11`; `RedHat 8`; `Amazon Linux 2 (rhel 7)`
i think you better to disable automatic updates, to avoid any issues. 
and update critical security bugs only. 

```
vi /etc/dnf/automatic.conf
upgrade_type = security
```
or disable it.
